catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceCallable.java

```diff
@@ -167,9 +165,9 @@ public void setInterruptDownload(boolean interruptDownload) {
 
     /**
      * Set to true when the current resource download should be canceled. Usually set by the
-     * @ReliableResourceInputStream when it is closed by the client.
      *
      * @param cancelDownload
+     * @ReliableResourceInputStream when it is closed by the client.
```

Comment:
Nope. That's how it was originally. I'm deleting it. Definitely seems out of place.